README.rst

```diff
@@ -77,13 +77,13 @@ PUDL is comprised of three core components:
     Python package is embedded with a set of of DOIs to indicate which version of the
     raw inputs it is meant to process. This process helps ensure that the ETL and it's
     outputs are replicable. You can read more about the ETL in the
-    `docs <https://catalystcoop-pudl.readthedocs.io/en/dev/intro.html#the-etl-process>`__.
+    `docs <https://catalystcoop-pudl.readthedocs.io/en/nightly/#the-etl-process>`__.
```

Comment:
Do we eventually want our docs, and datasette to by pointed at `stable`? I feel like most projects default to the latest stable version with the option to check out bleeding edge docs.

Reply:
Yeah, `stable` seems less surprising, but also `nightly` seems fine to me. We could maybe move it all to `latest` and then configure `latest` to point at `stable` via RTD - then if we want to switch things around we can.

Reply:
I think once we have successfully moved to an almost automated monthly release schedule, switching to stable seems fine.  But we have intended to do something like that many times, and I don't think having our default documentation be months out of date with what many people are using is a great idea.

Reply:
I agree. I think most people will gravitate towards `nightly` because it seems like the prefer freshness over stability.

I'm curious when people will point to `stable`. I feel like folks will either want the bleeding edge or a pinned version.